stdlib/asm/crypto/hashes/rpo.masm

```diff
@@ -32,12 +37,14 @@ end
 #! This requires that `end_addr=start_addr + 2n + 1`, otherwise the procedure will enter an infinite
 #! loop. `end_addr` is not inclusive.
```

Comment:
I'm not sure this is exactly correct. I think proving `end_addr = start_addr` also works fine.